compute/disks/createComputeHyperdiskPool.js

```diff
@@ -33,7 +33,7 @@ async function main() {
   // Project ID or project number of the Google Cloud project you want to use.
   const projectId = await storagePoolClient.getProjectId();
   // Name of the zone in which you want to create the storagePool.
-  const zone = 'europe-central2-b';
+  const zone = 'us-central1-a';
```

Comment:
We can't create storage pool in europe zone. Changed to US as it was before. The same problem was in Java directory: https://github.com/GoogleCloudPlatform/java-docs-samples/commit/e24337003c62b37f3894214849d6c38d9d831381